pkg/acsengine/const.go

```diff
@@ -33,3 +30,21 @@ const (
 	// PublicAgent represents the public agent node type
 	DCOSPublicAgent DCOSNodeType = "DCOSPublicAgent"
 )
+
+const (
+	KubernetesHyperkubeImageName         = "hyperkube-amd64:v1.5.1"
+	KubernetesDashboardImageName         = "kubernetes-dashboard-amd64:v1.5.1"
+	KubernetesExechealthzImageName       = "exechealthz-amd64:1.2"

```

Comment:
This looks good for now but might change in the future where ACS Engine will need to load/install different versions of each component based on overall Kubernetes orchestrator version. Just fyi.